scripts/publish.js

```diff
@@ -0,0 +1,23 @@
+const { exec } = require('shelljs');
+
+// validate git and npm
+require('./validate');
+
+// lint, unit test, and build
+exec('npm run lint && npm test && npm run build');
+
+// ensure bundle size is below threshold
+exec('npm run test:bundle');
+
+// run puppeteer tests
+exec('npm run test:e2e');
+
+// bump version
+const newVersionArg = process.argv[2] || 'patch';
```

Comment:
Are we unable to get the current version and automatically bump it?

Reply:
This is here to support what type of version bump to do. Similar to how [npm version](https://docs.npmjs.com/cli/version) takes in patch, minor, or major.